ui-src/src/drawing/layoutFormula.js

```diff
@@ -64,8 +64,14 @@ function mapHierarchyToPosition({ goal, hierarchy }, withHierarchies, screenWidt
 }
 
 export default function layoutFormula(screenWidth, goals, edges) {
+  // convert objects to arrays for iterating
```

Comment:
I moved these into here since they were ripe for refactoring, since we were just doing the same redundant work to convert outside this function in multiple places, before passing them into this function

Reply:
Yay :)